packages/suite/src/components/wallet/WalletLayout/index.tsx

```diff
@@ -60,10 +68,11 @@ const WalletLayout = (props: Props) => {
 
     return (
         <Wrapper>
+            {showEmptyHeaderPlaceholder && <EmptyHeaderPlaceholder />}
```

Comment:
do not forget about multiple returns in WalletLayout :)
also placeholder needs to be under all notifications otherwise it will push them down too!